modules/codemod/lib/v7/recategorizeIconButtons.ts

```diff
@@ -73,7 +74,19 @@ export default function transformer(file: FileInfo, api: API, options: Options)
 
```

Comment:
So it's not letting me suggest this properly, but we can type the above assignment to `variantProp` and simplify the code below. Line 72 to line 85 would be the following:
```
      const variantProp = attrs?.find(
        attr => attr.type === 'JSXAttribute' && attr.name.name === 'variant'
      ) as JSXAttribute | undefined;

      // Default IconButton variant is `circle`
      if (variantProp) {
        const valueBody = variantProp.value;
        if (valueBody?.type === 'JSXExpressionContainer') {
          variantProp.value = valueBody.expression as StringLiteral;
        }

        const variantPropValue = (variantProp.value as StringLiteral).value;

        (variantProp.value as StringLiteral).value = variantPropValue.replace(
```

Could you give that a try? I think we can do more to simplify the types but I want to start there.

Reply:
Just replaced lines 72-85, and see no issues/errors. 

Spec file isn't showing any issues either